src/error.rs

```diff
@@ -41,6 +43,12 @@ pub enum Error {
     /// Error when converting floating type to integer point value
     #[error("Floating type could not be converted to integer value")]
     FloatingPointToInteger,
+    /// Error if no input rule files are specified
+    #[error("No inputs were specified")]
+    NoInput,
+    /// Error if the user asked for an unimplemented feature
+    #[error("The specified input uses {0}, which is an unimplemented feature")]
+    Unimplemented(String),
```

Comment:
Please don't do stringly typed errors, have one variant for each feature (there are already some variants for unsupported features). Also, didn't we support multiple input files before?

Reply:
Yes, I was not sure if I should leave it this way, because it was just concatenating multiple files into a single string. Maybe this should just be left unimplemented until we get around to modules, but I can certainly also recreate the old behaviour.

Reply:
One drawback with the variant for each feature approach is that implementing a feature and removing that variant is a breaking change. But who cares anyway